contrib/test_jenkins.sh

```diff
@@ -850,8 +851,11 @@ run_mpi_tests() {
 
 		test_malloc_hooks_mpi
 
-		make_clean distclean
+		# Restore LD_LIBRARY_PATH so subsequent tests will not take UCX libs
+		# from installation directory
+		LD_LIBRARY_PATH=${save_LD_LIBRARY_PATH}
```

Comment:
Shouldn't we export the `LD_LIBRARY_PATH` here to fully revert the changes made above?
```suggestion
		export LD_LIBRARY_PATH=${save_LD_LIBRARY_PATH}
```